02-intermediate-ruby/lazy-with-let.md

```diff
@@ -33,19 +56,27 @@ describe Pet do
 end
 ```
 
-So in the above example the `before` block runs before each test and provides an instance variable `@pet` which we can use in our tests.  The problem with before blocks is that they **always** run before each test case.  Even if we don't want to use `@pet` in a few test cases, like for instance if we want to try creating a Pet with no arguments the before block will still run.
+In the above example the `before` block runs before each test and provides an instance variable `@pet` which we can use in our tests.  This can often serve as a helpful way to DRY testing code.
+
+Unfortunately `before` blocks **always** run before each test case.  Even if we don't want to use `@pet` in a few test cases, like for instance if we want to try creating a Pet with no arguments the before block will still run.
```

Comment:
Could you talk about the connection b/w `before` and `describe`? Specifically, the idea that if you have multiple `describe` blocks or nested `describe`s, you can control where it happens a little tighter.